components/x-gift-article/src/Form.jsx

```diff
@@ -15,7 +15,6 @@ export default (props) => (
 
```

Comment:
[Observation] this is regarding `<Title>` tag. In the original we have <Title title={props.title}> because there was only one prop - the title. My PR from years ago changed this to [unfurl all the props given](https://github.com/Financial-Times/x-dash/pull/579/files#diff-ba3498a6d99030b819f24b3f081537330bccc8791b31bbda9fceacda39df63e6R13) so I think we can go back to just picking out the title. But it's not blocking.